tests/examples/langgraph/personal_research_assistant/research_assistant.py

```
@@ -0,0 +1,249 @@
+import os
+import time
+import argparse
+from langgraph.graph import StateGraph, END
+from langchain_core.prompts import PromptTemplate
+from langchain_community.tools.tavily_search import TavilySearchResults
+from typing import TypedDict, Annotated, List, Dict, Any, Optional
```

Ruff (0.8.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)